web/src/styles/components/card.scss

```diff
@@ -28,3 +28,16 @@
   font-size: 0.88rem;
   font-weight: 600;
 }
+
+.usa-prose.text-white > h3 {
```

Comment:
pebble, lets chat through this, I don't think all three are needed

Reply:
done